include/xten/Conversion/Passes.td

```diff
@@ -36,4 +36,14 @@ def XTenToLinalg : Pass<"xten-to-linalg", "ModuleOp"> {
   ];
 }
 
+
+def XTenNNToTosa : Pass<"xten-nn-to-tosa", "ModuleOp"> {
```

Comment:
Any idea where subgraph is being decomposed? I suppose it doesn't belong here as it goes to `func.func`, implying this is a partial conversion - I think you should mention partial conversion in the description.

Reply:
We have the subgraph decomposition in our enterprise repo. Maybe we can move it here one day to mlir-xten?

But I will update the description to mention this is a partial conversion.